packages/react/.npmignore

```diff
@@ -33,3 +33,4 @@ visualtests
 !lib
 !lib-commonjs
 !lib-amd
+!dist
```

Comment:
`@fluentui/react` makes use of the `dist/` folder as well which was missed in last night's publishing so adding here